src/groupeddataframe/grouping.jl

```diff
@@ -373,9 +391,21 @@ function combine(f::Any, gd::GroupedDataFrame)
         return gd.parent[1:0, gd.cols]
     end
 end
+
 combine(gd::GroupedDataFrame, f::Any) = combine(f, gd)
-combine(gd::GroupedDataFrame, f::Pair...) = combine(f, gd)
-combine(gd::GroupedDataFrame, f::Pair) = combine(f, gd)
+
+function combine(gd::GroupedDataFrame, f::Union{Pair, AbstractVector{<:Pair}}...) 
+    vec_of_pairs = reduce(vcat, f)
+    combine(vec_of_pairs, gd)
+end
+
+function combine(gd::GroupedDataFrame, f::NamedTuple)
+    combine(f, gd)
+end
+
+function combine(gd::GroupedDataFrame, f::Tuple{Vararg{<:Pair}})
```

Comment:
Why not include this in the `Union` above? Also, use short function form.

Reply:
Its added, I forgot to delete.